xml/shared/src/main/scala/fs2/data/xml/internals/EventParser.scala

```diff
@@ -624,8 +625,7 @@ private[xml] object EventParser {
                   sb.append('>')
                 } else {
                   sb.append('&')
-                  for (i <- 0 until n)
-                    sb.append("gt;".charAt(i))
+                  sb.append("gt;".substring(0, n))
```

Comment:
I guess that's the only "real" change in this PR.

Reply:
Wow, why didn't I do it this way already? :facepalm: Thanks